docs/tremor-script/index.md

```diff
@@ -167,24 +167,87 @@ Record literals in `tremor-script` are syntactically equivalent to JSON document
 }
 ```
 
+#### Binary
+
+Binaries are based on the [Erlang bit syntax](https://erlang.org/doc/programming_examples/bit_syntax.html).
+
+Binary grammar:
+
+> ![record grammar](grammar/diagram/Binary.png)
+
+Binaryfields grammar:
+
+> ![fields grammar](grammar/diagram/BinaryFields.png)
+
+Binaryfield grammar:
+
+> ![field grammar](grammar/diagram/BinaryField.png)
+
+
+Parts of each field are: `<value>:<size>/<type>` where both `size` and `type` are optional.
+
+The types currently supported are:
+
+* `binary` - this can handle both binaries and strings, `size` here refers to the number of bytes
+* `integer` - this can represent integers, `size` here means size in bits. In addition the type can be prefixed with `big` and `little` for indianness and `singed` and `unsigned` for singedness.
```

Comment:
```suggestion
* `integer` - this can represent integers, `size` here means size in bits. In addition the type can be prefixed with `big` and `little` for indianness and `signed` and `unsigned` for signedness.
```

Reply:
:+1: